server/SocketManager/socketmanager.js

```diff
@@ -8,4 +10,37 @@ const removeSocket = usertoken => {
   delete sockets[usertoken];
 };
 
-export { sockets, updateSocket, removeSocket };
+const emitMessage = (socket, message, data) => {
+  if (socket) {
```

Comment:
The `emitMessage` function could benefit from additional error handling. Consider checking if the `message` and `data` parameters are valid before emitting. This will prevent potential runtime errors.
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*